models/GitHub.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: connor
+ * Date: 15/07/2016
+ * Time: 13:36
+ */
+
+namespace QA;
+
+class GitHub
+{
+    private $client_id;
+    private $client_secret;
+
+    function __construct($accessToken, $config)
```

Comment:
I'd say you can create a constructor that accepts a single object which contains all  the info. This is more flexible and future proof.  Don't forget to add  documentation to the constructor though.